README.md

```diff
@@ -6,14 +6,14 @@
 
 - [Quick Start](./docs/quick-start.md)
 
-Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers.
+Load Balancer Operator for Kubernetes is a Cluster API speaking operator for load balancers. It manages the lifecycle of the [Load Balancer and Ingress Service](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tree/master) resources for Kubernetes clusters.
 
 ## Features
 
 1. It reconciles Cluster API objects and provisions Service type LoadBalancer for control plane Machines to achieve HA.
-2. It leverages [Carvel Packaging APIs](https://carvel.dev/kapp-controller/docs/latest/packaging) to lifecycle manage load balancer provider operator. Currently, we support AKO([AVI Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes)) as a reference implementation.
+2. It leverages [Carvel Packaging APIs](https://carvel.dev/kapp-controller/docs/latest/packaging) to lifecycle manage load balancer provider operator. Currently, we support [NSX Advance LB Kubernetes Operator](https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes) as a reference implementation.
 3. It bridges [Cluster API](https://cluster-api.sigs.k8s.io/) and load balancer provider operator to ensure load balancer resources are cleaned up when cluster is deleted.
-4. For AKO, it also automates the user account creation and injection per cluster.
+4. For the NSX Advance Load Balancer, it also automates the user account creation and injection per cluster.
```

Comment:
For the NSX Advanced Load Balancer **operator**

Reply:
fixed